qiskit_nature/operators/second_quantization/qubit_converter.py

```diff
@@ -0,0 +1,111 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A converter from Second-Quantized to Qubit Operators."""
+from typing import List, Optional, Tuple, Union
+
+from qiskit.opflow import PauliSumOp
+from qiskit.opflow.primitive_ops import Z2Symmetries
+
+from qiskit_nature import QiskitNatureError
+from qiskit_nature.mappers.second_quantization import QubitMapper
+
+from . import SecondQuantizedOp
+
+# TODO two qubit reduction flag for parity mapping; special case it here? Or add some flag to
```

Comment:
I would add it here, as I think it is a reasonable user-convenience feature to have. Since it is a special case of the Z2 symmetries (which are handled here), this case should also be handled here and not on the mappers.
@pbark any comments?